backend/src/main/scala/cromwell/backend/standard/callcaching/RootWorkflowFileHashCacheActor.scala

```diff
@@ -48,7 +48,7 @@ class RootWorkflowFileHashCacheActor private[callcaching](override val ioActor:
           // hash to become available.
           cache.put(key, FileHashValueRequested(requesters = requester :: requesters))
         case FileHashSuccess(value) =>
-          sender() ! Tuple2(hashCommand.fileHashContext, IoSuccess(requester.ioCommand, value))
+          sender() ! Tuple2[Any, Any](hashCommand.fileHashContext, IoSuccess(requester.ioCommand, value))
```

Comment:
There was some change in 2.13.9 that gave us a lot of new warnings about constructions that resolve to `Any` or `Object`, which can be caused by programmer error.  considered finding some way to disable this check, but I think the changes that resolve it are actually neutral or improvements to the clarity of the code, and I ran out of places to fix before I ran out of energy to fix them.